docs/changelog/112665.yaml

```diff
@@ -0,0 +1,13 @@
+pr: 112665
+summary: Remove zstd feature flag for index codec best compression
+area: Codec
+type: enhancement
+issues: []
+highlight:
+  title: Enable ZStandard compression for indices with index.codec set to best_compression
+  body: |-
+    Before DEFLATE compression was used to compress stored fields in indices with index.codec index setting set to
+    best_compression, with this change ZStandard is used as compression algorithm to stored fields for indices with
+    index.codec index setting set to best_compression. Experiments have shown that ZStandard offers lower storage usage
+    (upto ~12%) and higher indexing throughput (upto 14%) compared to DEFLATE.
```

Comment:
should we also update the note on higher indexing throughput here?

Reply:
yes, I reworded this to: 58e48987deae850547b3cca1e04559e64aae6abd